autogen/main.tf.tmpl

```diff
@@ -111,7 +111,7 @@ resource "google_compute_backend_service" "default" {
   description                     = lookup(each.value, "description", null)
   connection_draining_timeout_sec = lookup(each.value, "connection_draining_timeout_sec", null)
   enable_cdn                      = lookup(each.value, "enable_cdn", false)
-  security_policy                 = lookup(each.value, "security_policy", var.security_policy)
+  security_policy                 = lookup(each.value, "security_policy", null) == null ? var.security_policy : each.value.security_policy
```

Comment:
Technically, this should be able to be reduced to the following, but it felt safer to me to do it the way that I currently have it.

```
  security_policy                 = lookup(each.value, "security_policy") == null ? var.security_policy : each.value.security_policy
```